.openhands/setup.sh

```
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# OpenHands runs this at workspace initialization — before session-level skill
+# discovery scans .agents/skills/. Submodules must be initialized by then, or
+# the gitlinks are empty directories and no SKILL.md is found.
+# An init failure (e.g. no network) must not kill the session: degrade
+# gracefully — already-initialized submodules keep working, uninitialized
+# ones just don't load this session.
+set -uo pipefail
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)